main/src/cgeo/geocaching/log/LogCacheActivity.java

```diff
@@ -360,7 +360,9 @@ public void onCreate(final Bundle savedInstanceState) {
         if (image == null) {
             image = Image.NONE;
         }
-        enablePostButton(false);
+        // TODO: Why is it disabled in onCreate? Probably it should be disabled only when
+        // there is some explicit issue.
+        //enablePostButton(false);
```

Comment:
This is disabled in onCreate as there is a relatively widely used connector (namely geocaching.com) which requires the (asynchronous) loading of some data in order to allow a successful post. So this behaviour is mandatory.

Reply:
Could you please advise how do I enable this button then? I tried to use simpler approach as in EC service, not using this loader class that I don't need. But onCreate was called after other method which was enabling the button

Reply:
@rsudev any update on this one? I believe proper solution should be to *enable* it by default and only if some connector needs additional data, it *may* disable this button.

I created #7188 for possible further steps and implemented dummy `LoaderManager` for `SuLogginManager`, as well as kept my comments here and in `ECLoggingManager`